runtime/testy/studentAPI.py

```diff
@@ -61,17 +62,16 @@ class Gamepad(StudentAPI):
     "joystick_right_y" : 3
   }
 
-  def __init__(self):
-    super().__init__(self.toManager, self.fromManager)
+  def __init__(self, toManager, fromManager):
```

Comment:
From a quick glance it seems to still preserve functionality, but I haven't actually tested it.